contracts/Vault2.sol

```diff
@@ -1,6 +1,26 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.4;
 
-contract Vault2 {
-    
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/utils/math/SafeMath.sol";
```

Comment:
This is not used. So we should remove it. Unused libraries cause additional gas during deployment.